scripts/tools/templater.js

```diff
@@ -53,6 +94,12 @@ async function copyTemplate(params) {
 
                 // Copy files that shouldn't be modified
                 if (config.get('coreSrc').length > 0) {
+
+                    //these functions should probably be spun off into a more robust solution
+                    //something that searches the VT2 sdk install folder for a list of required folders/files
+                    copyFolderRecursiveSync(sdkDir+"/streamable_resources/core", modDir)
```

Comment:
If there aren't a lot of files, maybe it's better to just copy them to the .template and .template-vmf folders? I doubt these will change again.

Reply:
I share you're doubts they will change again, but at the same time they did change and with the most recent update to SDK you can longer use the older 3XX physx files to build your mod. I think the safest option and the option for long term support would be to use whatever FS has bundled with the VT2 SDKTools.

This may be getting out of scope for VMB; I was also thinking about the need for some mods to pull in files from `streamable_resources/core/shader_nodes` and `streamable_resources/core/stingray_renderer` when making custom materials. There are some issues with custom materials that still need to be sorted out from Warrior Priest update, but custom materials will still require files from these folders.